src/core/mod.rs

```diff
@@ -7,6 +7,4 @@ pub mod operation_client;
 pub mod request_client;
 mod testing;
 
-pub use parsec_interface::operations::{psa_algorithm, psa_key_attributes};
-pub use parsec_interface::operations_protobuf::ProtobufConverter;
-pub use parsec_interface::requests::{Opcode, ProviderID};
+pub use parsec_interface;
```

Comment:
Maybe we can do:
```rust
pub use parsec_interface as interface;
```
So that it is shorter to import?
`use parsec_client::core::parsec_interface::requests::ProviderID` seems a bit redundant.

Reply:
I did think of that, but I'm not sure... Maybe it's a bit confusing as to what that module is (even as it is now).

Reply:
Yes, it would be very good to add some text in the doc of the [`core` module](https://docs.rs/parsec-client/0.1.0/parsec_client/core/index.html) to say that the interface is resurfaced at this level. I think that is important but you can choose to remove the `parsec_` or not.